packages/scheduler/src/Scheduler.js

```diff
@@ -336,6 +347,31 @@ function unstable_scheduleCallback(callback, deprecated_options) {
   return newNode;
 }
 
+function unstable_pauseExecution() {
+  isSchedulerPaused = true;
+}
+
+function unstable_continueExecution() {
+  isSchedulerPaused = false;
+  if (firstCallbackNode !== null) {
+    ensureHostCallbackIsScheduled();
+  }
+}
+
+function unstable_dumpQueue() {
+  var node = firstCallbackNode;
+  var callbacks = [];
+
+  if (node !== null) {
+    do {
+      callbacks.push(node);
+      node = node.next;
+    } while (node !== firstCallbackNode);
+  }
+
+  return callbacks;
+}
```

Comment:
Instead of exposing play, pause, and dump-queue– what if we just exposed the queue itself (`firstCallbackNode`) somewhere globally in DEV mode only? Then we could just write our own tiny helper function that read from it (if that was actually needed).

(This also assumes that we just use the browser's built-in play/pause script execution functionality.